snooty/postprocess.py

```diff
@@ -532,11 +533,34 @@ class TabsSelectorHandler(Handler):
     def __init__(self, context: Context) -> None:
         super().__init__(context)
         self.selectors: Dict[str, List[Dict[str, MutableSequence[n.Text]]]] = {}
+        self.stack: List[str] = []
+        self.scanned_pattern: List[str] = []
+
+    def scan_for_pattern(self, fileid_stack: FileIdStack, node: n.Node) -> None:
+        starting_point = 0
+        target_pattern = ["tabs", "tabs", "procedure"]
```

Comment:
@rayangler good question. I believe that could be the case, where it's not so much at which layer was the pattern discovered, but rather the pattern existed. Based on your comment, I assume this doesn't go beyond the second layer. I was under the impression, that having the logic in the `postprocess.py` would capture the pattern at any layer. Was my understanding incorrect?

Reply:
Oh no, you're correct. Please feel free to ignore this comment! I forgot that if there are anymore nested tabs, it should still fall under the target pattern since anything between the second `tabs` and the `procedure` is arbitrary

Reply:
No worries. It was still a good comment, as I was able to think over it again and both confirmed my assumption.